db/schema.rb

```diff
@@ -45,8 +45,8 @@
 
   create_table "games", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.jsonb "stats"
-    t.uuid "losing_team_id"
-    t.uuid "winning_team_id"
+    t.uuid "loser_id"
```

Comment:
Yeah, I like this. I think I initially wrote losing_team to be a little verbose with people not familiar with Rails.